### DISCUSSION TITLE
Clearing decision scope

### DISCUSSION THREAD

**Discussion by lihuijun11** [fossology/fossology#2087] [Q&A] [answered]
I click "Apply decision to all future occurrences of this file", and "Indentified" , and then clicke "submit"
Then, i want to recovery the default status, but i can not clear my decision.
How can I recovery the default status, so that when I upload the same file, the "Edited  Result" will be empty?

**Comment by GMishx** [comment, marked as answer]
Currently, there is only one way to remove the entry the entry with "Apply decisions to all future occurrences of this file" from DB.
You can list all such decisions with following query and remove entries which are not required. (Note: `scope = 1` => global, `scope = 0` => local).
```sql
SELECT * FROM clearing_decision WHERE scope = 1;
```

**Comment by YanickNoblanc** [comment]
Hello everyone,

I have been trying for several days to clear a decision on a license and enable 'Apply decision to all future occurrences of this file' so that all future uploads will be detected AND concluded as matching this license. Whatever I do, I can never manage to make it work as expected.
Here are the operations I have performed many times  :
- From Fossology server, go to Admin/License admin/Select License 
  - Select the license I created and modify MYLICENSE 'License Text'
  - Make sure Active=yes, Checked=yes, SPDX Compatible=yes, Detector type=Reference license, Conclusion=MYLICENSE, Reported license=MYLICENSE
- From a client machine scan (fossology REST API) a debian package containing a debian/copyright file containing the exacte  same 'License Text'
- From fossology server, go to browse, select the package that has been uploaded/scanned
- With license browser, display file debian/copyright
- Select Action=+, for MYLICENSE (at this stage it is not very clear from documentation whether Action='+' or 'X' should be selected, but whatever is selected, next problems remain the same).
- Enable ' Apply decision to all future occurrences of this file'
- Set Clearing decision type=Identified
- Click on 'Submit'
- From a client machine scan (fossology REST API) a second Debian package containing same license

I expect this new package to have MYLICENSE automatically detected AND concluded. But it is never the case:
- MYLICENSE is systematically detected
- But MYLICENSE is almost never 'Concluded'

Among the many tests I did, I did see MYLICENSE automatically concluded only a couple of times, without any clue on the reason it worked.

I initially had a server running fossology 3.11.0 and after more than 20 scans not automatically concluding , I tried upgrading to fossology 4.2.1, and did again many scans but the issue remains the same. MYLICENSE is almost never automatically concluded.

If someone could have hints on what is going wrong, I would be gratefull.
